app/donu/Schema.hs

```diff
@@ -555,3 +560,18 @@ instance JS.ToJSON DonuValue where
 
       unimplVal :: String -> JS.Value
       unimplVal str = typedPrim "string" str
+
+      -- Jupyter requires the LaTeX to be:
+      --
+      -- (1) Surrounded by $$ ... $$
+      -- (2) Using \\ to indicate newlines
+      --
+      -- This function performs surgery on the results of printLatex to conform
+      -- to these requirements.
+      printLatexEscaped :: Latex -> String
```

Comment:
anything jupyter specific should probably happen in the kernel instead of here

Reply:
Done.